app/components/PageIndicator.tsx

```diff
@@ -0,0 +1,66 @@
+import { tw, tx } from '@helpwave/common/twind/index'
+import type { PropsWithLanguage } from '@helpwave/common/hooks/useTranslation'
+import { useTranslation } from '@helpwave/common/hooks/useTranslation'
+import type { Languages } from '@helpwave/common/hooks/useLanguage'
+import ChevronLast from '../icons/ChevronLast'
+import ChevronLeft from '../icons/ChevronLeft'
+import ChevronFirst from '../icons/ChevronFirst'
+import { useState } from 'react'
+import ChevronRight from '../icons/ChevronRight'
+
+type PageIndicatorTranslation = {
+  of: string
+}
+const defaultOrganizationMemberListTranslations: Record<Languages, PageIndicatorTranslation> = {
+  en: {
+    of: 'of'
+  },
+  de: {
+    of: 'von'
+  }
+}
+
+export type PageIndicatorProps = {
+  initialPage: number, // starts with 0
+  numberOfPages: number,
+  onPageChanged: (page: number) => void
+}
+
+export const PageIndicator = ({
+  language,
+  initialPage,
+  numberOfPages,
+  onPageChanged
+}: PropsWithLanguage<PageIndicatorTranslation, PageIndicatorProps>) => {
+  const [currentPage, setCurrentPage] = useState(initialPage)
+  const translation = useTranslation(language, defaultOrganizationMemberListTranslations)
+
+  const changePage = (page:number) => {
+    setCurrentPage(page)
+    onPageChanged(page)
+  }
+
+  const onFirstPage = currentPage === 0
+  const onLastPage = currentPage === numberOfPages - 1
+  return (
+    <div className={tw('flex flex-row')}>
+      <button onClick={() => changePage(0)} disabled={onFirstPage}>
+        <ChevronFirst className={tx({ 'opacity-30': onFirstPage })}/>
+      </button>
+      <button onClick={() => changePage(currentPage - 1)} disabled={onFirstPage}>
+        <ChevronLeft className={tx({ 'opacity-30': onFirstPage })}/>
+      </button>
+      <div className={tw('flex min-w-[80px] justify-center mx-2')}>
+        <span className={tw('select-none text-right flex-1')}>{currentPage + 1}</span>
+        <span className={tw('select-none mx-2')}>{translation.of}</span>
+        <span className={tw('select-none text-left flex-1')}>{numberOfPages}</span>
```

Comment:
You don't need 3 elements for this, you can just use a single one and do something like this:
```tsx
<span className={tw('select none')}>{translation.pageXofY(currentPage + 1, numberOfPages)}</span>
```

Reply:
The issue isn't with the translation, but without three elements the numbers jump arround (the position changes) for different numbers which is annoying an distracts the user

Reply:
but that is still annoying if the text shifts, therefore I solved that. Why would we now move back to a solution which is 2 lines shorter but worse

Reply:
Because having this split into multiple parts is overly complicated and doesn't allow all that much flexibility while translating; as an example this doesn't allow converting the numbers to another representation such as `"page one of two"`. This flexibility isn't necessarily needed but you also should make the code more complicated as it needs to be when you can just rely on the inline layouting to do the job. Having this be 3 dom elements might even seem overly complicated and over-engineered to users

Reply:
I would evaluate not jumping text higher than reducing the number of dom elements by two, if the solution is still readable without much effort, which it currently is

If we later on encounter a situation where a single text would be better, we can still change it then without much effort